ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -188,6 +190,69 @@ public void SetTooltip(TFrameworkElement view, string tooltip)
             ToolTipService.SetToolTip(view, tooltip);
         }
 
+        /// <summary>
+        /// Sets the shadow color of the <typeparamref name="TFrameworkElement"/>.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="color">The shadow color.</param>
+        [ReactProp("shadowColor", CustomType = "Color")]
+        public void SetShadowColor(TFrameworkElement view, uint? color)
+        {
+            if (color.HasValue)
```

Comment:
Would there be any reason to track what props are set / unset to reclaim memory when shadow props are no longer set?

Reply:
Quite possibly, yes. Not sure how common that would be in practice, but certainly removing the ShadowHost if, eg, color is nulled would make sense, I think. Less sure about keeping track of the other properties, but I'm open to it.

I guess it also depends on expected default behavior though. Most of the props have default values on the XAML/Composition side, but does the user expect an element to get a shadow if they only set, say, offset? I don't really know what I'd expect myself in that scenario.

Reply:
The sibling element should now be properly removed/freed when, for example, `shadowColor` is set to `null`. The shadow is also correctly positioned if added after the view has been loaded on the XAML side.